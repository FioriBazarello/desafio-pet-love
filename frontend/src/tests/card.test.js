import React from 'react';
import { shallow } from 'enzyme';

import Card from '../components/card';

describe('Testing Card Component', () => {
  it('should be defined', () => {
    expect(Card).toBeDefined();
  });

  it('should render correctly', () => {
    const wrapper = shallow(<Card>Conte√∫do do Card</Card>);

    expect(wrapper).toMatchSnapshot();
  });
});