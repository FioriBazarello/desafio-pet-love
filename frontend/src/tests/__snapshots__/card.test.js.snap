// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Card Component should render correctly 1`] = `ShallowWrapper {}`;
